src/lv_conf_internal.h

```diff
@@ -580,7 +628,7 @@ e.g. "stm32f769xx.h" or "stm32f429xx.h"*/
 #endif
 
 /*Garbage Collector settings
- *Used if lvgl is bound to higher level language and the memory is managed by that language*/
+ *Used if lvgl is binded to higher level language and the memory is managed by that language*/
```

Comment:
These typos should be fixed in `lv_conf_template.h` as `lv_conf_internal.h` is auto generated.